### ISSUE TITLE
Operator unittests

### ISSUE THREAD

**Issue by LuisHsu** [ONNC/onnc-runtime#141]
Implement unittests in `unittest` branch

* [ ] Abs
* [ ] Acos
* [ ] Add
* [x] And
* [ ] ArgMax
* [ ] ArgMin
* [ ] Asin
* [ ] Atan
* [ ] AveragePool
* [ ] BatchNormalization
* [ ] Cast
* [ ] Ceil
* [ ] Clip
* [ ] Concat
* [ ] Constant
* [ ] Conv
* [ ] ConvTranspose
* [ ] Cos
* [ ] DepthToSpace
* [ ] Div
* [ ] Dropout
* [ ] Elu
* [ ] Equal
* [ ] Exp
* [ ] Flatten
* [ ] Floor
* [ ] GRU
* [ ] Gather
* [ ] Gemm
* [ ] GlobalAveragePool
* [ ] GlobalLpPool
* [ ] GlobalMaxPool
* [ ] Greater
* [ ] HardSigmoid
* [ ] Hardmax
* [ ] Identity
* [ ] InstanceNormalization
* [ ] LRN
* [ ] LSTM
* [ ] LeakyRelu
* [ ] Less
* [ ] Log
* [ ] LogSoftmax
* [ ] LpNormalization
* [ ] LpPool
* [ ] MatMul
* [ ] Max
* [ ] MaxPool
* [ ] MaxRoiPool
* [ ] Mean
* [ ] Min
* [ ] Mul
* [ ] Multinomial
* [ ] Neg
* [ ] Not
* [ ] Or
* [ ] PRelu
* [ ] Pad
* [ ] Pow
* [ ] RNN
* [ ] RandomNormal
* [ ] RandomNormalLike
* [ ] RandomUniform
* [ ] RandomUniformLike
* [ ] Reciprocal
* [ ] ReduceL1
* [ ] ReduceL2
* [ ] ReduceLogSum
* [ ] ReduceLogSumExp
* [ ] ReduceMax
* [ ] ReduceMean
* [ ] ReduceMin
* [ ] ReduceProd
* [ ] ReduceSum
* [ ] ReduceSumSquare
* [ ] Relu
* [ ] Reshape
* [ ] Selu
* [ ] Shape
* [ ] Sigmoid
* [ ] Sin
* [ ] Size
* [ ] Slice
* [ ] Softmax
* [ ] Softplus
* [ ] Softsign
* [ ] SpaceToDepth
* [ ] Split
* [ ] Sqrt
* [ ] Squeeze
* [ ] Sub
* [ ] Sum
* [ ] Tan
* [ ] Tanh
* [ ] Tile
* [ ] TopK
* [ ] Transpose
* [ ] Unsqueeze
* [ ] Upsample
* [ ] Xor
* [ ] experimental ATen
* [ ] experimental Affine
* [ ] experimental ConstantFill
* [ ] experimental Crop
* [ ] experimental GRUUnit
* [ ] experimental GivenTensorFill
* [ ] experimental If
* [ ] experimental ImageScaler
* [ ] experimental Loop
* [ ] experimental LoopIndexTensor
* [ ] experimental MeanVarianceNormalization
* [ ] experimental ParametricSoftplus
* [ ] experimental Scale
* [ ] experimental ScaledTanh
* [ ] experimental ThresholdedRelu
* [ ] ai.onnx.ml.ArrayFeatureExtractor
* [ ] ai.onnx.ml.Binarizer
* [ ] ai.onnx.ml.CastMap
* [ ] ai.onnx.ml.CategoryMapper
* [ ] ai.onnx.ml.DictVectorizer
* [ ] ai.onnx.ml.FeatureVectorizer
* [ ] ai.onnx.ml.Imputer
* [ ] ai.onnx.ml.LabelEncoder
* [ ] ai.onnx.ml.LinearClassifier
* [ ] ai.onnx.ml.LinearRegressor
* [ ] ai.onnx.ml.Normalizer
* [ ] ai.onnx.ml.OneHotEncoder
* [ ] ai.onnx.ml.SVMClassifier
* [ ] ai.onnx.ml.SVMRegressor
* [ ] ai.onnx.ml.Scaler
* [ ] ai.onnx.ml.TreeEnsembleClassifier
* [ ] ai.onnx.ml.TreeEnsembleRegressor
* [ ] ai.onnx.ml.ZipMap
